From 152387afadf3d189539e6c8bd4b6decc0e7f15f7 Mon Sep 17 00:00:00 2001
From: Vojtech Trefny <vtrefny@redhat.com>
Date: Wed, 7 Mar 2018 10:45:46 +0100
Subject: [PATCH] Do not try to update potfile during make all

---
 po/Makefile | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/po/Makefile b/po/Makefile
index 2f845cd..bbc6b5e 100644
--- a/po/Makefile
+++ b/po/Makefile
@@ -27,7 +27,7 @@ MOFILES		= $(patsubst %.po,%.mo,$(POFILES))
 PYSRC		= $(wildcard ../blivet/*.py ../blivet/*/*.py)
 SRCFILES 	= $(PYSRC)
 
-all::  update-po $(MOFILES)
+all::  refresh-po $(MOFILES)
 
 $(POTFILE): $(SRCFILES)
 	$(XGETTEXT) -L Python --keyword=_ --keyword=N_ --keyword=P_:1,2 $(SRCFILES)
@@ -65,5 +65,3 @@ install: $(MOFILES)
 	$(MSGFMT) -o $@ $<
 
 .PHONY: missing depend
-
-
-- 
1.8.3.1

